TOPICS.md

```diff
@@ -39,6 +40,8 @@
 | `hyades-vulnerability-analyzer-snyk-retry-store-changelog`<sup>1E</sup>                           | 3          | `cleanup.policy=compact`<br/>`segment.bytes=67108864`<br/>`max.compaction.lag.ms=0` |
 
 *<sup>1X</sup> The topic is subject to [co-partitioning requirements](#co-partitioning-requirements)*  
+*<sup>1D</sup> To use this notification, the PROJECT_VULN_ANALYSIS_COMPLETE group needs to be manually added through api call to the notify on condition
```

Comment:
Please use `<sup>3</sup>` instead. `1` is supposed to only be used for highlighting co-partitioning requirements.

Also please ensure the formatting is correct when rendered, at the moment it's kind of broken:

![image](https://github.com/DependencyTrack/hyades/assets/5693141/22fe3172-27d9-4a65-ae78-02b879edf927)